client/rest-high-level/src/test/java/org/elasticsearch/client/TransformIT.java

```diff
@@ -144,19 +151,36 @@ private void indexData(String indexName) throws IOException {
     }
 
     @After
-    public void cleanUpTransforms() throws Exception {
+    public void cleanUpTransformsAndLogAudits() throws Exception {
         for (String transformId : transformsToClean) {
-            highLevelClient().transform().stopTransform(
-                    new StopTransformRequest(transformId, Boolean.TRUE, null, false), RequestOptions.DEFAULT);
+            highLevelClient().transform()
+                .stopTransform(new StopTransformRequest(transformId, Boolean.TRUE, null, false), RequestOptions.DEFAULT);
         }
 
         for (String transformId : transformsToClean) {
-            highLevelClient().transform().deleteTransform(
-                    new DeleteTransformRequest(transformId), RequestOptions.DEFAULT);
+            highLevelClient().transform().deleteTransform(new DeleteTransformRequest(transformId), RequestOptions.DEFAULT);
         }
 
         transformsToClean = new ArrayList<>();
         waitForPendingTasks(adminClient());
+
+        // using '*' to make this lenient and do not fail if the audit index does not exist
```

Comment:
The same can be achieved with new `searchRequest.indicesOptions(IndicesOptions.LENIENT_EXPAND_OPEN)`

Reply:
`searchAll` does a index refresh which would fail if you give it an explicit index name.